libpod/runtime_img.go

```diff
@@ -51,7 +51,7 @@ func (r *Runtime) RemoveImage(ctx context.Context, img *image.Image, force bool)
 			imageCtrs = append(imageCtrs, ctr)
 		}
 	}
-	if len(imageCtrs) > 0 && len(img.Names()) <= 1 {
+	if len(imageCtrs) > 0 && (len(img.Names()) <= 1 || force && img.InputIsID()) {
```

Comment:
Will golang allow you to use parenthesis to make this line easier to disect?

Reply:
I added them, let's see if it complains